pyopenms_viz/_bokeh/core.py

```diff
@@ -541,11 +541,9 @@ class BOKEHMobilogramPlot(BOKEHChromatogramPlot, MobilogramPlot):
 
 
 class BOKEHSpectrumPlot(BOKEH_MSPlot, SpectrumPlot):
-    """
-    Class for assembling a Bokeh spectrum plot
-    """
+    def add_peptide_sequence(self, peptide_sequence: str, matched_fragments: List[Tuple[int, int]]):
```

Comment:
Please add a docstring here for good practice. Can be based on the docstring below and explain this is not implemented..